data/presets/building/annexe.json

```diff
@@ -0,0 +1,16 @@
+{
+    "icon": "maki-home",
+    "geometry": [
+        "area"
+    ],
+    "tags": {
+        "building": "annexe"
+    },
+    "terms": [
+        "annexe",
+        "adu",
```

Comment:
Other terms include “in-law unit” and “[secondary suite](https://en.wikipedia.org/wiki/Secondary_suite)”.

Reply:
thanks, added.
Ideally these terms would be translatable too.